### PR TITLE
No issue - Added privacy policy page

### PR DESCRIPTION
**Pull request by Manvel**

### CODE REVIEW

File: pages/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Privacy Policy
+permalink: privacy
+---
+
+## Analytics
+
+CBA is user centric product, CBA is only using Google Analytics to collecting
```
Comment (naarrek) [inline pages/privacy-policy.md]: CBA is using only Google Analytics

File: pages/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+minimal data only to understand how the product is used and find generic traffic
+pattern in chrome-automation.com website. No other data is collected and only
+used internally to understand how to improve the product. You can find summary
```
Comment (naarrek) [inline pages/privacy-policy.md]: used only internally

Comment (Manvel) [reply pages/privacy-policy.md]: As we would like to emphasize that it's only used internally and never shared with 3-rd parties. Guess using only in the beginning might be more explanatory in this case?

File: pages/privacy-policy.md (hunk 1 above)
Comment (Manvel) [reply pages/privacy-policy.md]: Used `CBA is user centric product, CBA is using Google Analytics only to collect minimal data, just to understand how the.` instead, as agreed. As it's more informative and makes clearer, that we collect minimal data *only*.

**Pull request merged**